services/m3dbnode/config/bootstrap.go

```diff
@@ -90,11 +90,14 @@ func (bsc BootstrapConfiguration) New(
 	opts storage.Options,
 	adminClient client.AdminClient,
 ) (bootstrap.ProcessProvider, error) {
+	if err := ValidateBootstrappersOrder(bsc.Bootstrappers); err != nil {
```

Comment:
mind adding some logging in the lambda where this is used - `server.go:kvWatchBootstrappers` - just to indicate what bootstrapper configuration we're continuing to use when this fails